Algorithm/QCAlgorithm.cs

```diff
@@ -3170,6 +3173,7 @@ private void SetLiveModeStartDate()
         /// Sets the statistics service instance to be used by the algorithm
         /// </summary>
         /// <param name="statisticsService">The statistics service instance</param>
+        [DocumentationAttribute(StatisticsTag)]
         public void SetStatisticsService(IStatisticsService statisticsService)
```

Comment:
Same comment for `SetStatisticsService/SetAlgorithmMode/SetDeploymentTarget` these are for lean engine to set only I think